pkgs/applications/window-managers/yabar/default.nix

```diff
@@ -29,6 +30,13 @@ stdenv.mkDerivation rec {
   postInstall = ''
     mkdir -p $out/share/yabar/examples
     cp -v examples/*.config $out/share/yabar/examples
+
+    ${lib.optionalString (configFile != null)
+      ''
+        wrapProgram "$out/bin/yabar" \
+          --add-flags "-c ${configFile}"
```

Comment:
It's a bit unusual to make command flags configurable in the package derivation, looking at the yabar man page it seems to read `~/.config/yabar/yabar.conf` by default so the `-c` flag is not mandatory.

Reply:
why though? You might want to configure the configuration file as a derivation (I do this ATM) and then it's placed in `~/.nix-profile` for instance.

Reply:
Fair enough, I don't have any strong feelings about it. The main downside I can think of is that it makes it tricky to use the binary cache when having a non-null `configFile` value.